app/src/main/java/org/greenstand/android/TreeTracker/utilities/Padding.kt

```diff
@@ -0,0 +1,18 @@
+package org.greenstand.android.TreeTracker.utilities
+
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+
+/**
+ * Simple padding values.
+ *
+ * Useful for supplying padding values
+ * without having to use a Modifier.
+ */
+data class Padding(
```

Comment:
It looks like padding modifiers have a very nice API. Why don't we just use them instead?

Reply:
I was looking for an api for that! I will do some digging and use that instead. Much prefer not to reinvent the wheel